test-with-jest/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "test-with-jest",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "clear": "rm -rf dist && rm -rf coverage",
+    "start": "node dist/mock-function/index.js",
+    "test": "jest --detectOpenHandles dist --coverage",
+    "build": "npm run clear && tsc",
+    "lint": "eslint \"./src/**/*.ts\" --fix",
+    "format": "prettier \"./src/**/*.ts\" --write",
+    "precommit": "npm run format && git add . && npm run lint"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@azure/cosmos": "^4.1.0",
+    "@azure/identity": "^4.4.1",
+    "dotenv": "^16.4.5",
+    "uuid": "^10.0.0",
+    "@faker-js/faker": "^8.4.1"
+  },
+  "devDependencies": {
+    "@types/jest": "^29.5.12",
+    "@types/node": "^22.4.0",
+    "@types/uuid": "^10.0.0",
+    "@typescript-eslint/eslint-plugin": "^8.2.0",
+    "eslint": "^8.56.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-jest": "^28.8.0",
+    "husky": "^9.0.10",
+    "jest": "^29.7.0",
+    "prettier": "^3.2.4",
+    "typescript": "^5.5.4"
+  },
+  "husky": {
```

Comment:
We can probably remove all the husky stuff as well - pre-commit hooks can be painful to manage

Reply:
ok